tests/e2e_suite.py

```diff
@@ -166,7 +166,7 @@ def filter_fun(x):
         elapsed -= test_elapsed
 
     if not args.dry_run:
-        network.stop_all_nodes(verbose_verification=False)
+        network.stop_all_nodes(skip_verification=True)
```

Comment:
PR says we're disabling this in LTS compatibility, but this line also disables in suite. Deliberate?